src/Contract Test Generator.postman_collection.json

```diff
@@ -1259,7 +1272,8 @@
 									"        }        \r",
 									"    }\r",
 									"    else {\r",
-									"        pm.response.to.have.status(400);\r",
+									"        const statusCode = pm.response.code\r",
+									"        pm.expect(statusCode === 400 || statusCode === 422).to.be.true;\r",
```

Comment:
Before, the code was saying: 
"Some tests expect a failure, and these failures are systematically 400 errors"

But now this is saying: 
"Some tests expect a failure, and these failures are systematically 400 errors or 422 errors"